metapool/tests/test_prep.py

```diff
@@ -285,101 +331,6 @@ def test_remove_qiita_id(self):
         obs = remove_qiita_id('project_')
         self.assertEqual(obs, 'project_')
 
-    def test_get_run_prefix(self):
```

Comment:
why remove tests vs. fixing them?

Reply:
Ah, I should admit that my new get_run_prefix() isn't truly a drop-in replacement for the old one. The old one takes four parameters including the project-name, the sample-id, the lane-number and a general file-path. All of those parameters were being fed into that same glob wildcard string parameter that we discussed yesterday. Because of what I'm now doing up-front - globbing once and doing all this checking and confirmation, there's no need to glob for every sample and no need to pass all of those parameters. Instead of looking for files to match sample-ids, we want to invert it and find all of the files and try to match sample-ids to them. That lets us know when some fastqs failed to map to any sample-id and will become lost data. Once we do that the below tests are redundant.

Reply:
Got it, I like the idea of removing old code and tests and replace them with better ones.